Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityRaycastHit.cs

```diff
@@ -35,7 +35,7 @@ public MixedRealityRaycastHit(bool raycastValid, RaycastHit hitInfo)
                 triangleIndex = hitInfo.triangleIndex;
                 textureCoord = hitInfo.textureCoord;
                 textureCoord2 = hitInfo.textureCoord2;
-                transform = hitInfo.transform;
+                transform = hitInfo.collider.transform;
```

Comment:
Maybe add a comment here to explain why it is necessary to use .collider.transform instead.